TestREADME.md

```diff
@@ -55,7 +55,7 @@ Example Response body:
 "{CTUTR":"1234567890}"
 ```
 
-### POST /test-only/cross-regime/register/:identifier
+### POST /test-only/cross-regime/register/GRS
```

Comment:
Maybe underneath the Request header a few lines below mention that a valid regime must be sent as a query parameter. Example of using this:
/test-only/cross-regime/register/GRS?grsRegime=VATC